### ISSUE TITLE
root failed to get data: a bytes-like object is required, not 'str' fping.py:100

### ISSUE THREAD

**Issue by DblD** [20c/vaping#51]
### Steps to reproduce : 
`git clone git@github.com:20c/vaping.git `
`cd ./vaping`
`docker build -t vaping . && docker run --rm -it -p7021:7021 vaping`

### Result :
`vaping.plugins.fping popen fping -u -C10 -p20 -e 8.8.8.8 4.2.2.1 208.67.222.222 __init__.py:67
root b'8.8.8.8        : 183.93 183.50 183.60 184.01 184.04 183.46 183.92 183.65 207.85 183.63\n' fping.py:68
root failed to get data: a bytes-like object is required, not 'str' fping.py:100
root b'4.2.2.1        : 143.44 145.34 142.94 143.97 143.33 143.31 143.34 143.17 143.45 143.83\n' fping.py:68
root failed to get data: a bytes-like object is required, not 'str' fping.py:100
root b'208.67.222.222 : 27.77 19.95 19.53 61.96 19.96 19.96 19.79 20.04 19.92 21.53\n' fping.py:68
root failed to get data: a bytes-like object is required, not 'str' fping.py:100
vaping.plugins.fping sending __init__.py:151
2018-11-17 13:13:35,716 - vodka - DEBUG: Empty data row ignored.
vodka Empty data row ignored. handlers.py:100
2018-11-17 13:13:35,716 - vodka - DEBUG: Empty data row ignored.
vodka Empty data row ignored. handlers.py:100
2018-11-17 13:13:35,716 - vodka - DEBUG: Empty data row ignored.
vodka Empty data row ignored. handlers.py:100`

**Comment by vegu** [comment]
I pushed a potential fix for this at 9fd5f5b - as your error message is somewhat different than the one i was able to reproduce, can you confirm on your end if this fixed your problem as well, please.

**Comment by DblD** [comment]
@vegu Still brakes with same error for me.

**Comment by vegu** [comment]
@DblD alright, thanks - what os and python version are you running?

**Comment by DblD** [comment]
> @DblD alright, thanks - what os and python version are you running?

Loading your project docker file, which is set for `FROM python:3.6`

**Comment by vegu** [comment]
I am not overly familiar with docker, but just to make sure, did you pip install from github to apply my fix above? Just pulling the change into your local repo wont be enough since - from what i can tell - docker installs vaping from pypi which has not received the patch yet.

```
pip install git+https://github.com/20c/vaping.git -U
```

If you did already do that, then any more information you can give about your setup would be helpful.

**Comment by DblD** [comment]
> I am not overly familiar with docker, but just to make sure, did you pip install from github to apply my fix above?

Yes, I've made silly assumption that Dockerfile was for development, so when I pulled and started docker,  I got the same issue. But then realized that it was still same code on line 120 fping.py that was on the release branch. 

When I get spare cycle ill create development docker for this project. It will be easier to participate if all are on same configuration.
